extensions/documents/__init__.py

```diff
@@ -62,15 +71,28 @@ def register(self):
                 description = desc,
                 color = discord.Colour.from_rgb(r=255,g=192,b=60)
             )
+            embed.set_author(
+                name="Synixe Rules",
+                url="https://github.com/Synixe/Documents/blob/master/SynixeRules.pdf",
+                icon_url=self.bot.user.avatar_url
+            )
             await message.channel.send(embed=embed)
         #except:
         #    await message.channel.send("There was an error finding that rule, it most likely doesn't exist.")
 
     async def rules(self, args, message):
+        embed = discord.Embed(
+            color = discord.Colour.from_rgb(r=255,g=192,b=60)
+        )
```

Comment:
move this below the `return` line, you are creating an embed and throwing it away. Only make the embed if we are going to need it